spec/deaf_grandma_spec.rb

```diff
@@ -9,10 +9,27 @@
   end
   
   it "says 'NOT SINCE 1964!' when we yell" do
-    #implement your test here
+    expect(script.speak("HI GRANDMA")).to eq "NOT SINCE 1964!"
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
-    #implement your test here
+    2.times { script.speak("BYE") }
+
+    expect(script.speak("BYE")).to eq "SEE YOU LATER SONNY!"
+  end
+
+  it "resets after saying SEE YOU LATER SONNY" do
+    2.times { script.speak("BYE") }
+    expect(script.speak("BYE")).to eq "SEE YOU LATER SONNY!"
+    expect(script.speak("BYE")).to eq "NOT SINCE 1964!"
   end
+
```

Comment:
Ideally, each test should cover a single aspect of what the code does.  If you want to test the reset functionality, why not just go for `3.times` in the setup?  If you want to test the 'SEE YOU LATER' string, then what is the last test for?